docs/getting-started-provider-dev.md

```diff
@@ -145,10 +145,22 @@ and versioning information.
 
 ### Getting and Building Cloud Provider OpenStack
 
-Following the [GitHub Workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md)
-guidelines for Kubernetes development, set up your environment and get the latest development repository. Begin
-by forking both the Kubernetes and Cloud-Provider-OpenStack projects into your GitHub into your local
-workspace (or bringing your current fork up to date with the current state of both repositories).
+Following the [GitHub Workflow](https://github.com/kubernetes/community/blob/master/contributors/guide/github-workflow.md) guidelines for Kubernetes development, set up your environment and get the latest development repository. Begin by forking both the Kubernetes and Cloud-Provider-OpenStack projects into your GitHub into your local workspace (or bringing your current fork up to date with the current state of both repositories).
+
+`make` will build, test, and package this project. This project uses [go dep](https://golang.github.io/dep/) for dependency management.
+
+If you don't have a Go Environment setup, we also offer the ability to run make
```

Comment:
could we add this part as a separate heading?? it is little deviating from the flow.

Reply:
ok